src/software/field_tests/ball_placement_field_test.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+import software.python_bindings as tbots_cpp
+import sys
+import math
+from proto.ssl_gc_common_pb2 import Team
+from proto.ssl_gc_state_pb2 import Command
+from proto.import_all_protos import *
+from software.field_tests.field_test_fixture import *
+
+from software.simulated_tests.simulated_test_fixture import *
+from software.logger.logger import createLogger
+from software.simulated_tests.robot_enters_region import RobotEventuallyEntersRegion
+from proto.message_translation.tbots_protobuf import create_world_state
+
+logger = createLogger(__name__)
+
+
+def test_ball_placement(field_test_runner):
+    # robot ID
+    id = 2
+    # point to place ball
+    placement_point = tbots_cpp.Point(0, 0)
+
+    world = field_test_runner.world_buffer.get(block=True, timeout=WORLD_BUFFER_TIMEOUT)
+    print("Here are the robots:")
+    print(
+        [
+            robot.current_state.global_position
+            for robot in world.friendly_team.team_robots
+        ]
+    )
```

Comment:
Lets improve this by dynamically picking the robot id based on the friendly robot that's in the world, which is closest to the ball.

Instead of printing the robot positions, we should print the robot ids that we see.

Reply:
Since the field test is just running referee commands, the tactic assignment should do that, having the ID parameter there doesnt make sense in the first place